indev/libinput_drv.h

```diff
@@ -43,6 +49,22 @@ typedef enum {
   LIBINPUT_CAPABILITY_TOUCH    = 1U << 2
 } libinput_capability;
 
+typedef struct {
+  int fd;
+  struct pollfd fds[1];
+
+  int button;
+  int key_val;
+  lv_point_t most_recent_touch_point;
+
+  struct libinput *libinput_context;
+  struct libinput_device *libinput_device;
+
+#if USE_XKB
+  xkb_drv_state xkb_state;
+#endif /* USE_XKB */
+} libinput_drv_state;
```

Comment:
Usually we use `_t` suffix for `tyepdef`ed types.

Reply:
Ah, good call. Made the change.